src/iris/api.py

```diff
@@ -162,6 +162,7 @@ def ts_to_sql_datetime(ts):
     'created': 'UNIX_TIMESTAMP(`incident`.`created`) as `created`',
     'owner': '`target`.`name` as `owner`',
     'current_step': '`incident`.`current_step` as `current_step`',
+    'title_variable_name': '`template_variable`.`name` as `title_variable_name`'
```

Comment:
Looks like you're adding title_variable_name into the fields that we get from a query to `/incidents`. I like this idea, but let's take it a step further and actually inject a new field into the JSON that we return, like `title`. Then, the API can be responsible for grabbing the title from the context (if it exists), and the clients can just use that field without needing to dig into the context.

Reply:
/incidents actually already returns the title information in the "title_variable_name" variable. It's defined in the incident_columns.

Reply:
Sure, but I mean that we can have the API abstract away the title variable name and just return the incident title. E.g returning `{id: 123, title: "5xx Errors above threshold"...}` instead of `{id: 123, title_variable_name: "name", context: {"name": "5xx Errors above threshold"}}`.

Reply:
Changed.